### PR TITLE
cleaned up some stuff

### PR DESCRIPTION
**Pull request by Gav06**

### CODE REVIEW

File: src/main/java/me/srgantmoomoo/Main.java [hunk 1]
```diff
@@ -118,9 +118,9 @@ public void init (FMLInitializationEvent event) {
 		log.info("postman initialization finished.");
 	
 	}
-	
+
 	@EventHandler
 	public void postInit (FMLPostInitializationEvent event) {
-		
+		Display.setTitle(Reference.NAME + " " + Reference.VERSION);
```
Comment (moomooooo) [inline src/main/java/me/srgantmoomoo/Main.java:124]: wouldn't this be better in preInit 0_0

File: src/main/java/me/srgantmoomoo/postman/api/event/EventProcessor.java [hunk 2]
```diff
@@ -41,6 +41,8 @@
 
 	public EventProcessor() {
 		instance = this;
+		Main.EVENT_BUS.subscribe(this);
```
Comment (moomooooo) [inline src/main/java/me/srgantmoomoo/postman/api/event/EventProcessor.java:44]: what was the point of moving this lol

File: src/main/java/me/srgantmoomoo/postman/api/mixin/mixins/MixinEntityPlayerSP.java [hunk 3]
```diff
@@ -1,29 +1,30 @@
...
-import me.srgantmoomoo.postman.api.event.Event.Era;
-import me.srgantmoomoo.postman.api.event.events.PlayerMotionUpdateEvent;
-import me.srgantmoomoo.postman.api.event.events.PlayerMoveEvent;
-import me.srgantmoomoo.postman.api.event.events.PlayerUpdateEvent;
-
 @Mixin(EntityPlayerSP.class)
 public abstract class MixinEntityPlayerSP extends AbstractClientPlayer {
 
+
+	// poo
```
Comment (moomooooo) [inline src/main/java/me/srgantmoomoo/postman/api/mixin/mixins/MixinEntityPlayerSP.java:22]: cap

File: src/main/java/me/srgantmoomoo/postman/client/module/modules/player/ChatSuffix.java [hunk 4]
```diff
@@ -18,18 +23,20 @@ public ChatSuffix() {
...
-	@SubscribeEvent
-	public void onChat(final ClientChatEvent event)
-	{
-		if(toggled) {
-		for (final String s : Arrays.asList("/", ".", "-", ",", ":", ";", "'", "\"", "+", "\\", "@"))
-		{
-			if (event.getMessage().startsWith(s)) return;
-		}
-		event.setMessage(event.getMessage() + " " + "\u23D0" + toUnicode(" " + Reference.NAME + " strong")); 
+	@EventHandler
```
Comment (moomooooo) [inline src/main/java/me/srgantmoomoo/postman/client/module/modules/player/ChatSuffix.java:26]: thanks, forge event 😒

**Comment by moomooooo** [comment]
un merged.... broke chat event i believe and keyevent maybe, clickgui doesn't open lol and certain things like esp do not work, and commands (prolly cause of chat event).

**Comment by Gav06** [comment]
:(

**Comment by Gav06** [comment]
failed contributor

**Pull request merged**